.circleci/config.yml

```diff
@@ -1386,7 +1386,31 @@ jobs:
       - checkout
       - attach_workspace:
           at: /tmp/workspace
-      - run: git clone --depth 1 https://github.com/nomiclabs/hardhat.git
+      - run:
+          name: Ensure pnpm is installed if npm is present
+          command: |
+            if command -v npm &> /dev/null; then
+              sudo npm install -g pnpm
+            fi
```

Comment:
Can `npm` even be missing here? The job is hard-coded to `cimg/node:18.16` image so I think we can assume we have it.

Still, weird that the "node" image does not have `pnpm` installed by default these days.

Reply:
Ah, wait, the pnpm thing is separate thing from from switching to the latest release, isn't it? I.e. it's the fix while switching to a release is a separate CI improvement?Why is it even in the same PR? It should be at least in a separate commit.